src/api/v1beta1/properties.go

```diff
@@ -137,6 +137,40 @@ func (dk *DynaKube) ActiveGateImage() string {
 	return fmt.Sprintf("%s/linux/activegate:latest", registry)
 }
 
+// EecImage returns the Extension Controller image to be used with the dk DynaKube instance.
+func (dk *DynaKube) EecImage() string {
+	if dk.ActiveGateMode() {
```

Comment:
shouldn't this check if the statsd capability is used?

Reply:
It was needed only to make sure there is an `ActiveGate` spec to look up the image in. I replaced it with `.NeedsStatsd()` anyway for consistency.